prime/src/util/Health.ts

```diff
@@ -1,9 +1,15 @@
+export const HEALTH_LOW_COLOR = 'rgba(235, 87, 87, 1)';
+export const HEALTH_MEDIUM_COLOR = 'rgba(242, 201, 76, 1)';
+export const HEALTH_HIGH_COLOR = 'rgba(0, 193, 67, 1)';
+export const HEALTH_LOW_THRESHOLD = 1.05;
+export const HEALTH_MEDIUM_THRESHOLD = 1.5;
```

Comment:
This was 1.25 before. 1.5 is pretty high I think.

Reply:
I updated it to match the health bar, which turns yellow at around 1.5

Reply:
It's more of a greenish yellow at that point though. This is straight up yellow

Reply:
What if we met in the middle at around 1.35?

Reply:
sgtm